src/state/machines/turn.ts

```diff
@@ -1099,6 +1099,7 @@ export const TurnMachine = setup({
               ({ context, event }) => TurnMachineGuards.notPlayedCard({ context }, event.card.uid),
               ({ context, event }) => TurnMachineGuards.ownsCard({ context }, event.card.uid),
               ({ context, event }) => TurnMachineGuards.notDisasterCard({ context }, event.card),
+              ({ context, event }) => TurnMachineGuards.notLitterCard({ context }, event.card),
```

Comment:
I think we need this on the move ability as well, not sure if we should be able to move the litter card, but I doubt it